Common/Statistics/Statistics.cs

```diff
@@ -331,6 +331,7 @@ public class Statistics
                     { "Expectancy", Math.Round((winRate * averageWinRatio) - (lossRate), 3).ToStringInvariant() },
                     { "Net Profit", Math.Round(totalNetProfit * 100, 3).ToStringInvariant() + "%"},
                     { "Sharpe Ratio", Math.Round(SharpeRatio(listPerformance, riskFreeRate), 3).ToStringInvariant() },
+                    { "SortinoRatio", Math.Round(SortinoRatio(listPerformance, riskFreeRate), 3).ToStringInvariant() },
```

Comment:
nit - should be "Sortino Ratio" ?
We will need to update all the regression stats to add this I believe?

---
Should test an optimization too, take a look at `OptimizationBacktestJsonConverter` think it will need an update?